tests/test_bedrock_authentication.py

```diff
@@ -0,0 +1,177 @@
+import os
```

Comment:
I'm not sure this test adds value - crewAI doesn't ship boto3, the user provides it. I think this test can be disregarded